### PR TITLE
[PHP] Add PHP 8.3

### PR DESCRIPTION
**Pull request by Ayesh**
Follow-up to #196185, which added PHP 8.4.

This PR requests to add PHP 8.3.

Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/196728)

### CODE REVIEW

File: manifests/p/PHP/PHP/8/3/8.3.14/PHP.PHP.8.3.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Created with PHPWatch/winget-pkgs - https://github.com/PHPWatch/php-winget-manifest
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
```
Comment (mdanish-kh) [inline manifests/p/PHP/PHP/8/3/8.3.14/PHP.PHP.8.3.installer.yaml:2]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
```

File: manifests/p/PHP/PHP/8/3/8.3.14/PHP.PHP.8.3.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Created with PHPWatch/winget-pkgs - https://github.com/PHPWatch/php-winget-manifest
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
```
Comment (mdanish-kh) [inline manifests/p/PHP/PHP/8/3/8.3.14/PHP.PHP.8.3.locale.en-US.yaml:2]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
```

File: manifests/p/PHP/PHP/8/3/8.3.14/PHP.PHP.8.3.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with PHPWatch/winget-pkgs - https://github.com/PHPWatch/php-winget-manifest
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
```
Comment (mdanish-kh) [inline manifests/p/PHP/PHP/8/3/8.3.14/PHP.PHP.8.3.yaml:2]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.9.0.schema.json
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Version fix for the schema headers

**Comment by Ayesh** [comment]
Thank you @mdanish-kh, you are right I had missed it. Fixed and force-pushed.

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**